packages/node/src/cli.js

```diff
@@ -285,6 +285,13 @@ const main = async (args = hideBin(process.argv)) => {
         global: true,
         group: BEHAVIOR_GROUP,
       },
+      'scuttle-global-this': {
+        describe: 'Enable scuttling of globalThis',
+        type: 'boolean',
+        global: true,
+        group: BEHAVIOR_GROUP,
+        coerce: (value) => value === true ? {enabled: true} : value,
```

Comment:
Would prefer to just keep this as a boolean for the sake of the CLI and we can coerce it later

```suggestion
        coerce: Boolean,
```

Reply:
fixed 937e97f3b